dist/images/ovnkube.sh

```diff
@@ -67,6 +67,7 @@ fi
 # OVN_SSL_ENABLE - use SSL transport to NB/SB db and northd (default: no)
 # OVN_REMOTE_PROBE_INTERVAL - ovn remote probe interval in ms (default 100000)
 # OVN_EGRESSIP_ENABLE - enable egress IP for ovn-kubernetes
+# OVN_UNPRIVILEGED_MODE - execute CNI ovs/netns commands from host (default off)
```

Comment:
Shouldn't the default be "on"? Hm... isn't NVIDIA currently using `--unprivileged-mode`? How are you configuring it?

Also, `OVN_SSL_ENABLE` uses `yes`/`no` and maybe this should do the same for consistency

Reply:
thanks for the feedback and review! Yes indeed will change to yes/no to be consistent across.. this feature is to enable the case when openvswitch is installed via container..

Reply:
Yes, we use --unprivileged-mode by default. Downstream, in ovnkube.sh, we are directly use --unprivileged-mode.